icu4c/source/test/cintltst/cstrtest.c

```diff
@@ -215,7 +215,7 @@ TestInvariant() {
         log_err("u_charsToUChars(variantChars) failed\n");
     }
 
-#ifdef NDEBUG
+#ifndef U_DEBUG
```

Comment:
I think the logic in the comment below might need to be updated, as the way it is written would sort of imply:  `#ifndef U_DEBUG`

Reply:
Nevermind, I see you fixed this in another commit. :)